vagrant/bootstrap_scripts/arch-sanitizers-gcc.sh

```diff
@@ -49,11 +49,12 @@ rm -fr "$BUILD_DIR"
 # See:
 #   https://bugzilla.redhat.com/show_bug.cgi?id=1827338#c3
 #   https://github.com/systemd/systemd-centos-ci/issues/247
-export LDFLAGS="-Wl,--no-as-needed"
 
 meson "$BUILD_DIR" \
       --werror \
       -Dc_args='-fno-omit-frame-pointer -ftrapv' \
+      -Dcpp_args='-fno-omit-frame-pointer -ftrapv' \
+      -Db_asneeded=false `# See the FIXME (--as-needed) above` \
```

Comment:
These line appears to break `scorecard`: https://github.com/ossf/scorecard/pull/1297 :-)

Reply:
That's a nice trick though :-)

Reply:
Hah, that's unfortunate. Should I drop it or is this fixable on the scorecard's side?

Reply:
`scorecard` uses https://github.com/mvdan/sh and ideally it should be fixed there. As far as I can tell, that's exctly what was reported in https://github.com/mvdan/sh/issues/636

I'd probably keep it :-)

Reply:
Ah, sweet, thanks for looking into it :)